clarifying that DBMS privileges do not get backed up (#1501)
neo4j/docs-operations: modules/ROOT/pages/backup-restore/online-backup.adoc

- `all` - include both `roles` and `users`.
- `none` - does not include any metadata.
[NOTE]
====

> Privileges which only pertain to the DBMS (rather than the database being backed up), are not included in the backup. For example, `GRANT ROLE MANAGEMENT ON DBMS TO $role` would not be backed up.

Reviewer: ```suggestion
Privileges specific to the DBMS and not to the backed-up database are not included in the backup. 
For instance, `GRANT ROLE MANAGEMENT ON DBMS TO $role` will not be backed up.
```

Accordingly, `roles` and `users` that do not have database-related privileges are not included in the backup (e.g. those with only DBMS or no privileges).

> It is recommended to use `SHOW USERS`, `SHOW ROLES` and `SHOW ROLE $role PRIVILEGES` to get the complete list of users, roles and privileges in these situations.

Reviewer: ```suggestion
It is recommended to use `SHOW USERS`, `SHOW ROLES`, and `SHOW ROLE $role PRIVILEGES` to get the complete list of users, roles and privileges in these situations.
```
